README.md

```diff
@@ -58,123 +56,136 @@ See [contributing](CONTRIBUTING.md) for more information.
 
-*romcal's code logic is developed according to calendar requirements descibed in various church documents sourced from the internet (and even from Wikipedia). If you notice discrepancies between romcal's output and actual liturgical dates, please do contribute your fixes or submit an issue on GitHub.*
+_Calendar entries for this module are pulled from various sources from the net. As such their accuracy cannot be ensured. If you find an incorrect calendar entry (e.g. wrong date, wrong feast type, spelling issue, typos), you are most welcome to contribute to the source code or inform me so that the necessary changes can be made to make this a more robust and reliable module._
+
+_romcal’s code logic is developed according to calendar requirements descibed in various church documents sourced from the internet (and even from Wikipedia). If you notice discrepancies between romcal’s output and actual liturgical dates, please do contribute your fixes or submit an issue on GitHub._
 
 ### Testing romcal
+
 romcal employs `TDD` using `mocha` and `should`.
 Run `npm test` in your console to view test output.
 
-`Travis CI` is used to validate romcal builds to ensure functionality is working as expected.
+Travis CI is used to validate romcal builds to ensure functionality is working as expected.
 
 ## Builds <a name="builds"></a>
 
-Running `npm run build` in the romcal root directory will generate a packaged version of romcal suitable for direct use in a browser.
+The romcal source code is actually the same for backend and frontend usage. To make it working on the frontend side, the frontend library is wrapped and built thanks to `webpack`. This build step is done automatically when you perform a `npm install` on the romcal directory.
 
-- `dist/romcal.bundle.min.js` A minified and obfruscated bundle of romcal + all its dependencies wrapped in a UMD module shell that is suitable for being included directly in browsers
+But if you have edited the codebase, you can update manually the frontend library with this command:
+
+```
+npm run build
+```
+
+The frontend library is generated in the `dist` folder, not included in the romcal codebase. When built, it contains `romcal.bundle.min.js`, a minified and obfruscated bundle of romcal + all its dependencies wrapped in a UMD module shell that is suitable for being included directly in browsers.
 
 ## Usage <a name="usage"></a>
 
-> romcal has been re-written using ES6, and use the new import/export syntax to manage node modules. However, thanks to `esm` and `babel`, it can still be included by other non ES6 aware node modules via the normal CommonJs `require` call.
+> :information_source: romcal was re-written using ES6, and use the new import/export syntax to manage node modules. However, thanks to `esm` and `babel`, it can still be included by other non ES6 aware node modules via the normal CommonJs `require` call.
```

Comment:
you r stuck on this form which which is less correct than the previous one, but never mind

Reply:
Sorry, I misunderstood you. Did you wanted it to be `romcal has been re-written`?

Reply:
No, let's save our energy for other things 😉
And sorry to have been a bit impatient these last days.
Thanks Tukusej’s!